openhsi/cameras.py

```diff
@@ -148,23 +150,23 @@ def __init__(self, **kwargs):
                 "ModuleNotFoundError: No module named 'arena_api'.", stacklevel=2
             )
 
-        #init api and connect to device
-#         try:
-#             self.arsys.device_infos
-#             print("2")
-#             if self.settings["mac_addr"]:
-#                 device_infos={"mac": self.settings["mac_addr"]} # use specfic camera
-#                 print("3")
-#             else:
-#                 device_infos={} # or use first camera found
+        # init api and connect to device
+        #         try:
+        #             self.arsys.device_infos
+        #             print("2")
+        #             if self.settings["mac_addr"]:
+        #                 device_infos={"mac": self.settings["mac_addr"]} # use specfic camera
+        #                 print("3")
+        #             else:
+        #                 device_infos={} # or use first camera found
 
-#             # self.device = arsys.create_device(device_infos=[{"mac": mac_addr}])[0]
+        #             # self.device = arsys.create_device(device_infos=[{"mac": mac_addr}])[0]
 
-#             self.device = arsys.create_device(device_infos=[device_infos])[0]
-#         except:
-#             warnings.warn(
-#                 "DeviceNotFoundError: Please connect a lucid vision camera and run again.",
-#                 stacklevel=2)
+        #             self.device = arsys.create_device(device_infos=[device_infos])[0]
+        #         except:
+        #             warnings.warn(
+        #                 "DeviceNotFoundError: Please connect a lucid vision camera and run again.",
+        #                 stacklevel=2)
```

Comment:
Don't think this should be commented out. I'll make changes to import error checking.

Reply:
I found these commented out and it was causing docs to fail because of the indentation.

Reply:
hmm, its showing as a change though in your branch, right?  Can we try rolling back the whole LucidCamera block to start of branch?

Reply:
These comments are also in the start of the branch
https://github.com/openhsi/openhsi/blob/a54b6fe166d0a45147a128a60a91ecb452181413/openhsi/cameras.py#L151
but also in you chrisdev branch before that
https://github.com/openhsi/openhsi/blob/276d292efa4bd2ac89c98ab8f4b8ca282ce18edd/openhsi/cameras.py#L173

Reply:
oh right,I see, my bad, that was an old version that should be removed. lines 173 (151) to 189 (167) can actually be removed if you want.